client/my-sites/woocommerce/woop/landing-page.tsx

```diff
@@ -28,7 +27,7 @@ interface Props {
 
 const images = [ { src: Image01 }, { src: Image02 }, { src: Image03 }, { src: Image04 } ];
 
-const WoopLandingPage: React.FunctionComponent< Props > = ( { startSetup, siteId } ) => {
+const WoopLandingPage: React.FunctionComponent< Props > = ( { siteId } ) => {
```

Comment:
Should `startSetup` be removed from [the calling code](https://github.com/Automattic/wp-calypso/blob/29a481a4009a868f5ad85741b7709130edccf8fb/client/my-sites/woocommerce/dashboard/required-plugins-install-view.jsx#L583) as well?

Reply:
Nice suggestion 🙇‍♂️. Addressed [here](https://github.com/Automattic/wp-calypso/pull/59785/commits/4ed4425a0c8059b9b192c37e8ad8f7db8274a2f5)